packages/codemirror-lang-chordpro/src/index.ts

```diff
@@ -2,24 +2,22 @@ import {parser} from "./syntax.grammar"
 import {LRLanguage, LanguageSupport, indentNodeProp, foldNodeProp, foldInside, delimitedIndent} from "@codemirror/language"
 import {styleTags, Tag, tags as t} from "@lezer/highlight"
 
-export const customTags = {
-  chord: Tag.define(), // define custom tag, that can be picked up by the style configuration
-}
-
 export const ChordProLanguage = LRLanguage.define({
   parser: parser.configure({
     props: [
       styleTags({
-        Chord: customTags.chord,
-        MetaData: t.meta,
-        Comment: t.comment,
-        Bracket: t.squareBracket,
-        String: t.string,
+        Chord: t.keyword,
+        Comment: t.lineComment,
+        DirectiveName: t.attributeName,
+        DirectiveValue: t.attributeValue,
+        Lyric: t.content,
+        "{ }": t.brace,
+        "[ ]": t.squareBracket,
```

Comment:
This scenario `[]` only highlights the first bracket.

Reply:
As soon as there's a letter in between it picks it up.

![CleanShot 2022-09-01 at 08 55 33](https://user-images.githubusercontent.com/23046374/187959057-acc2604b-2285-4dbb-84d5-d18e6571aee2.png)

![CleanShot 2022-09-01 at 08 55 46](https://user-images.githubusercontent.com/23046374/187959114-5c833096-7654-4945-b445-a13dba7d5c8d.png)

Reply:
Fixed in 5370e38